packages/cli/tests/cli/validation/call-handler-with-tuple/generated/schema.ts

```diff
@@ -29,10 +29,6 @@ export class MyEntity extends Entity {
     }
   }
 
-  static loadInBlock(id: string): MyEntity | null {
-    return changetype<MyEntity | null>(store.get_in_block("MyEntity", id));
-  }
-
```

Comment:
weird that tests remove this 🤔

Reply:
since you didn't had latest CLI change built locally that is why it was removing this https://github.com/graphprotocol/graph-tooling/pull/1296/commits/7e94f469844519d0d8cc088074c2f19da1edb261